libiocage/cli/start.py

```diff
@@ -54,6 +54,7 @@ def cli(ctx, rc, jails):
             ctx.parent.print_events(jail.start())
 
         except Exception:
+            raise
```

Comment:
This shouldn't be here. Unless you intend to never use `failed_jails` and continue in the loop.